.github/scripts/income-limits-data-sync.rb

```diff
@@ -11,11 +11,11 @@
 
 # Define csv files hash with table names as the index.
 files = {
-  "sdsadm.std_zipcode": "std_zipcode_temp.csv",
-  "sdsadm.std_state": "std_state_temp.csv",
-  "sdsadm.std_incomethreshold": "std_incomethreshold_temp.csv",
-  "sdsadm.std_gmtthresholds": "std_gmtthresholds_temp.csv",
-  "sdsadm.std_county": "std_county_temp.csv",
+  "sdsadm.std_zipcode": "std_zipcode.csv",
```

Comment:
These changes make the GHA write the canonical CSVs that are later consumed by the Sidekiq import scripts.